### PR TITLE
Added Kitty terminal emulator

### PR DESCRIPTION
**Pull request by bdmorin**
Added Kitty, pretty much the same thing as Alacritty, but ready to go on multiple platforms.

## Checklist for submitting a pull request to `Terminals Are Sexy`:

- [x] I have carefully **read and comply** with the [Contributing Guidelines](https://github.com/k4m4/terminals-are-sexy/blob/master/CONTRIBUTING.md) of this repo.
- [x] I have **searched** the [PRs](https://github.com/k4m4/terminals-are-sexy/pulls) of this repo and believe that this is not a duplicate.

 

- **kitty**: 
- **https://sw.kovidgoyal.net/kitty/**: 
- **Terminal Emulation Applications**: 
- **the fast, featureful, GPU based, terminal emulator**:

### CODE REVIEW

**Comment by bdmorin** [comment]
apparently I didn't see someone already submitted kitty, my bad. I'll delete.

File: readme.md [hunk 1]
```diff
@@ -126,6 +126,7 @@ Check out my [blog](https://nikolaskama.me/) and follow me on [Twitter](https://
 * [Go2Shell](http://zipzapmac.com/Go2Shell) - Opens a terminal window to the current directory in Finder.
 * [Cathode](http://www.secretgeometry.com/apps/cathode/) - Vintage terminal for macOS.
 * [Alacritty](https://github.com/jwilm/alacritty) - Cross-platform, GPU-accelerated terminal emulator.
+* [Kitty](https://sw.kovidgoyal.net/kitty/) - Cross-platform, GPU-accelerated terminal emulator.
```
Comment (kovidgoyal) [inline readme.md]: Just FYI, kitty is very much not the same thing as alacritty. Just run your eye over the list of major features here: https://sw.kovidgoyal.net/kitty

**Comment by DArtagan** [comment]
@k4m4 May we resuscitate this pull-request?  The kitty they're talking about here still isn't included.

Confusingly, there is:
* "KiTTY" (included in the list): Fork from version 0.70 of PuTTY.
* "kitty": Cross-platform, GPU-accelerated terminal emulator.

It would be could to include the latter as well.

**Comment by k4m4** [comment]
@DArtagan, thank you for mentioning this. @bdmorin, could you please squash the two commits into one? Thanks.

**Comment by k4m4** [comment]
Closing due to inactivity.

**Comment by okkdev** [comment]
still missing...

**Pull request closed without merging**